MAUI/TabView/Header-Display-Mode.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Header Display Mode in Maui Tabbed View control | Syncfusion
+description: Learn here all about Header Display Mode support in Syncfusion Maui Tabbed View (SfTabView) control and more.
```

Comment:
Learn here all about header display mode support in Syncfusion .NET MAUI Tab View (SfTabView) control and more.

Reply:
Updated the description